src/main/kotlin/io/github/moulberry/notenoughupdates/util/SkyBlockTime.kt

```diff
@@ -82,7 +82,7 @@ data class SkyBlockTime(
                 else -> "Undefined!"
             }
 
-            val name = when (month / 4 + 1) {
```

Comment:
I'm probably gonna revert this commit cause it is in my unit test changes already.

Reply:
🆗